native/cocos/ui/videoplayer/VideoPlayer-ios.mm

```diff
@@ -182,6 +187,21 @@ - (void)setKeepRatioEnabled:(BOOL)enabled {
         self.playerController.videoGravity = AVLayerVideoGravityResize;
 }
 
+- (void)setMute:(BOOL)enabled {
+    [self.playerController.player setMuted:enabled];
+}
+
+- (void)setLoop:(BOOL)enabled {
+    _keepLoopEnabled = enabled;
+    [[NSNotificationCenter defaultCenter]addObserver:self selector:@selector(runLoopTheVideo:) name:AVPlayerItemDidPlayToEndTimeNotification object:self.playerController.player.currentItem];
```

Comment:
I think only enabled needs to add play end listener.

Reply:
yes. you're right.